### PR TITLE
(Release 4.0.0): HDS-2069: Select part 5

### PR DESCRIPTION
**Pull request by NikoHelle**
## Description

Added keyboard commands, focus shifting and handling, onFocus/onBlur and active-descendant for accessibility.

Added also Playwright-tests and PW utils.

Missing:
- theming
- storage for making easier for users to memoize props.
- docs

## Related Issue

Closes [HDS-2069](https://helsinkisolutionoffice.atlassian.net/browse/HDS-2069)

## How Has This Been Tested?

Unit and Playwright

## Demos:

Links to demos are in the comments

## Add to changelog

Changelog added later with last part

[HDS-2069]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-2069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/react/src/components/select/Select.stories.tsx [hunk 1]
```diff
@@ -588,5 +497,124 @@ export const VirtualizedSingleselectWithoutGroups = () => {
...
+          .blurred {
+            padding: 10px;
+          }
+
+          .focused {
+            background-color: #defcde;
+          }
+
+          .blurred {
+            background-color: #ececec;
```
Comment (laurakarhu) [inline packages/react/src/components/select/Select.stories.tsx:569]: We should not put any non-brand-colors to examples, as others might consider it as allowed color. Hds color-black-10 would be #e6e6e6

File: packages/react/src/components/select/Select.stories.tsx [hunk 2]
```diff
@@ -588,5 +497,124 @@ export const VirtualizedSingleselectWithoutGroups = () => {
...
+    <>
+      <style>
+        {`
+          .focused,
+          .blurred {
+            padding: 10px;
+          }
+
+          .focused {
+            background-color: #defcde;
```
Comment (laurakarhu) [inline packages/react/src/components/select/Select.stories.tsx:565]: $color-tram-medium-light | #a3e3c2 would be similar HDS color

**Review by laurakarhu — APPROVED** [review]
Very focused. Minor remarks on the example colors, those can be closed after changes.

File: packages/react/src/components/select/Select.stories.tsx (hunk 2 above)
Comment (NikoHelle) [reply packages/react/src/components/select/Select.stories.tsx:565]: Good idea, I'll change these.

**Pull request merged**